test/functional/api/server_notifications_spec.lua

```diff
@@ -78,13 +79,7 @@ describe('notify', function()
   end)
 
   it('cancels stale events on channel close', function()
-    if isCI() then
-      pending('hangs on CI #14083 #15251')
-      return
-    elseif helpers.skip_fragile(pending) then
-      return
-    end
-    if helpers.pending_win32(pending) then return end
+    skip(isCI(), 'hangs on CI #14083 #15251')
```

Comment:
Nice! How does this work without the `return`, though?

Reply:
I think returns are only needed when used outside of `it`, not inside. I think this is true even for `pending`. It's in my todo list to investigate and to correct the test readme.